docs/tutorials/graphql-service.rst

```diff
@@ -78,7 +78,88 @@ The equivalent in our service is:
 Getting Points from a Spec
 --------------------------
 
-TODO
+Most importantly, is the ability to obtain a list of scan points from a Spec. 
+GraphQL gives the user the ability to request one or more fields from an object, 
+allowing them to obtain data that is relevant only to their application.
+
+The 'getPoints' query makes use of this, giving users the ability to select from
+one or more of the following fields:
+
+- numFrames: the total number of frames produced by the Spec
+- returnedFrames (WIP): the number of frames returned, limited by the maxPoint arguement
+- axes: a list of axes present in the Spec and its associated scan points
+
+Within axes:
+- axis: a list of the axes present in the Spec
+- lower: a list of lower bounds that are each present in a frame
+- midpoints: a list of midpoints that are each present in a frame
+- upper: a list of upper bounds that are each present in a frame
+
+Within lower, middle and upper:
+- string: returns the requested points as a string
```

Comment:
Probably best to say this is the numpy formatted string, which truncates, so is a human readable summary rather than machine readable

Reply:
updated to "returns the requested points as a human readable numpy formatted string"